src/modules/user/user.service.ts

```diff
@@ -198,9 +201,18 @@ export class UserService {
     }
   }
 
-  async addUserInfo(payload: UpdateUserDto, user: UserDto): Promise<void> {
+  async addUserInfo(payload: AddUserInfoDto, user: UserDto): Promise<void> {
     try {
-      await this.updateUserByEmail(user.email, payload);
+      if (payload.skills) {
+        const { skills, ...payloadNoSkills } = payload;
+        const userSkills = await this.propertyService.convertIdToSkills(skills);
+        const currentUser = await this.findByEmail(user.email);
```

Comment:
find other way please, make full query to get user data if you have it its too expensive. you already you have user

Reply:
fixed. deleted method from properties.service and replaced it with RelationQueryBuilder. 
![image](https://user-images.githubusercontent.com/95277626/204097096-23557ac4-2a7a-4cf2-a431-6d207d089d78.png)
-the first query one we need to make sure that when user re-enter skills we delete it from the table, before adding new skills. if there is not first query, it will add user skills to the table and not delete them